app/zadanie02.js

```diff
@@ -1 +1,23 @@
-//Twój kod
\ No newline at end of file
+//Twój kod
+console.log('Witam');
+let myTimeout1 = setTimeout(()=>{
```

Comment:
Działać - działa, ale dwie rzeczy
1. Czy jakiś pomysł jak to zautomatyzować :) ?
2. `const` będzie tu lepszy - nigdy nie potrzebujemy zmieniać raz dodanego timeout id.

Reply:
Dzięki :) już poprawiam rozwiązanie